cli/tenzir_platform/helpers/oidc.py

```diff
@@ -147,22 +155,18 @@ def _device_code_flow(self) -> dict[str, str]:
                 break
             elif token_data["error"] not in ("authorization_pending", "slow_down"):
                 print(token_data["error_description"])
-                authenticated = True
+                # authenticated = True
+                break
```

Comment:
Raise `PlatformCLIFatalError` exception (or just `exit(-1)` until then if this is a lot of effort)